test/unit/HeadersTest.php

```diff
@@ -193,8 +193,7 @@ public function testRemoveLangWithCustomDomainPattern()
             array('/zh/chs/', '/', 'zh-CHS'),
         );
         $env = array(
-            'HTTP_HOST' => 'testsite.com',
-            'REQUEST_URI' => '/zh/'
```

Comment:
I found my mistake about test.
I wanted to test, when accessed by zh lang.
So, I should do `array('zh-hant-hk.com', 'testsite.com', 'zh-Hant-HK')` not `array('zh-hant-hk.com', 'zh-hant-hk.com', 'zh-Hant-HK')` in testRemoveLangWithCustomDomainPattern.
And in this case, `'REQUEST_URI' => '/zh/'` is needed.

Reply:
I think the only way we can be sure about these tests is to include REQUEST_URI, HTTP_HOST in the test cases.
```
// only path
array('/zh/', '/zh/', 'zh-Hant-HK'),
```
This one for example. `/zh/` should become `/` if HTTP_HOST was `zh-hant-hk.com`, otherwise `/zh/`. I think we should be testing this.

Reply:
Yes. It is better to set some $_SERVER env per test.
I felt annoying, so I set one REQUEST_URI :stuck_out_tongue_closed_eyes: